misc/stdlib_diff/myers.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"slices"
```

Comment:
> [thehowl](https://github.com/thehowl) [on Dec 13, 2023](https://github.com/gnolang/gno/pull/1425#discussion_r1425454096)
seeing as you are importing slices, please add a go.mod file in this directory putting the go version at go 1.21

fixed on eaf6a87b2bcf0dc7f61662ef23b9bc80cc1a26d6